kernel_gateway/gatewayapp.py

```diff
@@ -148,16 +153,16 @@ def _prespawn_count_default(self):
 
     default_kernel_name_env = 'KG_DEFAULT_KERNEL_NAME'
     default_kernel_name = Unicode(config=True,
-        help='''The default kernel name to use when spawning a kernel (KG_DEFAULT_KERNEL_NAME env var)''')
+        help="""The default kernel name to use when spawning a kernel (KG_DEFAULT_KERNEL_NAME env var)""")
     def _default_kernel_name_default(self):
         # defaults to Jupyter's default kernel name on empty string
         return os.getenv(self.default_kernel_name_env, '')
 
     list_kernels_env = 'KG_LIST_KERNELS'
     list_kernels = Bool(config=True,
-        help='''Enables listing the running kernels through /api/kernels
+        help="""Enables listing the running kernels through /api/kernels
```

Comment:
~~Enables~~Permits listing of the running kernels ~~through~~ using API, i.e. /api/kernels